tests/checkpoint/test_checkpoint.py

```diff
@@ -1450,9 +1460,300 @@ def test_newstyle_checkpoint_instantiates_and_produces_a_validation_result_with_
     assert validation_result.meta["checkpoint_name"] == checkpoint_name
 
 
+@pytest.mark.slow  # 1.15s
+def test_newstyle_checkpoint_raises_error_if_batch_request_and_validator_are_specified_in_constructor(
```

Comment:
Can we please add `@pytest.mark.integration`?

Reply:
@cdkini There are no integration tests marked in this module -- they are all in `CLI`.  Doing so universally feels out of scope.  Thanks.

Reply:
@cdkini I took care of it -- thanks.